### DISCUSSION TITLE
How to switch species

### DISCUSSION THREAD

**Discussion by 1024488648** [GMOD/jbrowse-components#2028] [Q&A] [answered]
The jbrowse I'm using is the version embedded into the web page.I would like to switch species in my web page,Is there any way to do that？

**Comment by cmdcolin** [comment, marked as answer]
Hi there,
In general the embedded version only handles a single species at a time. You would have to implement the logic for switching species yourself e.g. by creating a new view model with the new species

**Reply by shen-y-eric** [reply]
Sorry if this is a dumb question (I'm a beginner), but do you mind elaborating a little more on "creating a new view model" for the new species? I thought that I would be able to just add another species' genome assembly to <code>assembly.js</code>, and it would allow me to choose from the assemblies on the web page.

**Reply by garrettjstevens** [reply]
When you say the embedded version, I'm assuming you mean the UMD build that you use in a script tag, like in the [tutorial](https://jbrowse.org/jb2/docs/bcc2020_embedding_jbrowse_03_simple_site).

The full JBrowse Web app does allow you to have multiple assemblies and choose which one to use. The embedded version, however, is what we call the Linear Genome View, which is one of the views you can use in the full JBrowse Web app. It's meant for displaying a single assembly at a time and does not contain the logic for handling multiple assemblies, since that is part of the full app, so you would need to add it yourself. For example, you might have to call `new JBrowseLinearGenomeView()` again with the assembly and tracks for the new assembly when the user chooses it.

**Reply by shen-y-eric** [reply]
I see thank you for the help!
